src/ccextractor.c

```diff
@@ -122,7 +122,7 @@ int api_start(struct ccx_s_options api_options)
             switch (ctx->total_inputsize)
             {
                 case -1*ENOENT:
-                    fatal(EXIT_NO_INPUT_FILES, "Failed to open input file: File does not exist.");
+                    fatal(EXIT_NO_INPUT_FILES, "Failed to open one or more input file(s): File does not exist.");
```

Comment:
Is this accurate? It doesn't seem possible that we can fail to open more than once since we are fatal()'ing on the first failure.

Reply:
You're right, I am sorry. Updated to `Failed to open one of the input file(s): File does not exist.`